### PR TITLE
Presets: fix a bug when saving freezes the progress dialog (because of the \r in the CLI text)

### PR DESCRIPTION
**Pull request by limonspb**
Bug noted by @haslinghuis

The bug:
if you try to "load backup" from the file with `\r\n` instead of `\n` it might freeze the saving progress dialog.
It happens because the firmware does not reply to the empty CLI commands with only `\r` and the callback of data response is not being called.

This PR fixes this bug by **not** sending the lines that contain only `\r` to CLI engine.
[test_CRLF.txt](https://github.com/betaflight/betaflight-configurator/files/8313561/test_CRLF.txt)

### CODE REVIEW

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> PASS

**Pull request merged**
